src/botPage/view/Dialogs/Dialog.js

```diff
@@ -8,7 +10,9 @@ export default class Dialog {
         this.componentId = `${id}-component`;
 
         ReactDOM.render(
-            <DialogComponent id={this.componentId} title={title} content={content} options={options} />,
+            <Provider store={store}>
+                <DialogComponent id={this.componentId} title={title} content={content} options={options} />
+            </Provider>,
             $(`#${id}`)[0]
```

Comment:
in order to remove jquery from project we can use `document.getElementById` here
```suggestion
            </Provider>,
            document.getElementById(id)
```

Reply:
Sure yea :+1: